src/App.js

```diff
@@ -1,20 +1,27 @@
 import React, { Component } from 'react'
-import movieData from './movieData'
 import Movies from './Movies'
 import SingleMovie from './SingleMovie'
+import { fetchData } from './APIcalls'
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies: { movieData },
+      movies: '',
       singleMovie: ''
```

Comment:
Same goes for single movie, only I'd imagine its going to be an object eventually